docs/self-hosted-server/minikube.mdx

```diff
@@ -107,6 +113,30 @@ Then, update your local repository with the following command:
 $ helm repo update
 ```
 
+### Setup MongoDB pod and service
```

Comment:
How about moving this section before `Add yorkie-team Helm chart in your local repository` section?

Also, we can exclude `and it avoids errors during Yorkie server provisioning.`.